en/docs/administer/product-security/logins-and-passwords/working-with-encrypted-passwords.md

```diff
@@ -214,9 +237,11 @@ If you start the WSO2 API Manager as a background job, you will not be able to p
         * For Linux: The file name should be `password-persist`
         * For Windows: The file name should be `password-persist.txt`
 
-2.  Add the primary keystore password (which is `wso2carbon` by default) to the new file and save. By default, the password provider assumes that both private key and keystore passwords are the same. If not, the private key password must be entered in the second line of the file.
+2.  Add the primary keystore password (which is `wso2carbon` by default) to the new file and save it. 
+
+     By default, the password provider assumes that both the private key and keystore passwords are the same. If not, the private key password must be entered in the second line of the file.
```

Comment:
"that both the private key and keystore passwords", shall we change this to "that both the private key password and the keystore passwords"

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/789/commits/9ba49a0403a2a0d620b949ed2475a8cb46a66b87